Integrations/integration-Quinc.yml

```diff
@@ -0,0 +1,341 @@
+configuration:
+- display: ""
+  name: ServerName
+  defaultvalue: ""
+  type: 0
+  required: true
+- display: ""
+  name: token
+  defaultvalue: ""
+  type: 0
+  required: true
+- display: ""
+  name: scheme
+  defaultvalue: http
+  type: 15
+  required: true
+  options:
+  - http
+  - https
+script:
+  script: |-
+    import sys, os, requests, datetime, json
+    from requests.exceptions import HTTPError
+
+    # disable insecure warnings
+    requests.packages.urllib3.disable_warnings()
+
+    AUTH_HEADERS = { 'EnterpriseApiKey' : demisto.params()['token'], 'Accept': 'application/json', 'Content-Type':  'application/json'}
+    USE_SSL = not demisto.params().get('insecure', True)
```

Comment:
you use `insecure` param but you haven't defined it in the params. please define it in the params